models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -438,7 +446,27 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     if ("leafGrowth" %in% pft.names) {
       param[which(param[, 1] == "leafGrowth"), 2] <- pft.traits[which(pft.names == "leafGrowth")]
     }
-  }  ## end loop over PFTS
+
+    #update LeafOnday and LeafOffDay
+    if (settings$model$leaf_phenology){
```

Comment:
Having both a model tag and a run$inputs tag seems unnecessarily complicated and isn't backwards compatible
```suggestion
    if (!is.null(settings$run$inputs$leaf_phenology)){
```